features/nanostack/coap-service/test/coap-service/unittest/coap_connection_handler/test_coap_connection_handler.c

```diff
@@ -394,8 +394,11 @@ bool test_socket_api_callbacks()
     memset(&buf, 1, 16);
 
     socket_callback_t *sckt_data = (socket_callback_t *)malloc(sizeof(socket_callback_t));
-    memset(sckt_data, 0, sizeof(socket_callback_t));
-
+    if(sckt_data)
```

Comment:
I opened this file, similar conditions have this coding style:

```
bool test_coap_connection_handler_send_data()
{
    coap_security_handler_stub.counter = -1;
    if (-1 != coap_connection_handler_send_data(NULL, NULL, ns_in6addr_any, NULL, 0, false)) {
        return false;
    }
```

Please update

Reply:
@0xc0170 ,sorry for long time not reply when I work hard in my job, these codes were there before this pr, and  you can  look at  this coding style in some other files in mbed os test code, is it need to change?